--- a/src/internal/sys_interface/icollector.cpp
+++ b/src/internal/sys_interface/icollector.cpp
@@ -151,8 +151,8 @@ namespace lsvpd
 			fillMe->addDeviceSpecific( key, "Microcode Build Date",
 							val, 90 );
 		else if ( key == "ME" )
-			fillMe->addDeviceSpecific( key, "Microcode Entitlement "
-							"Date", val, 90 );
+			fillMe->addDeviceSpecific( key, "Update Access Key Exp"
+						   " Date", val, 90 );
 		else if( key[ 0 ] == 'U' )
 			fillMe->addUserData( key, "User Data", val, 90, true );
 		else if( key[ 0 ] == 'Z' )
